llvm/lib/Passes/PassBuilder.cpp

```diff
@@ -660,7 +661,8 @@ Expected<HardwareLoopOptions> parseHardwareLoopOptions(StringRef Params) {
       int Count;
       if (ParamName.getAsInteger(0, Count))
         return make_error<StringError>(
-            formatv("invalid HardwareLoopPass parameter '{0}' ", ParamName).str(),
+            formatv("invalid HardwareLoopPass parameter '{0}' ", ParamName)
```

Comment:
This seem like unrelated formatting changes that should probably be reverted from this patch?

Reply:
Right. Reverted it. Thanks.